### PR TITLE
Amend content for Unused material form link

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
Amend content for Unused material form link

#### Why
BA and designer feedback

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1409,7 +1409,7 @@ en:
         clar_page_intro_html: |
           If needed, please complete an
           <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Special Prep Forms">appropriate special prep form</a>
-          and/or <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">Unused materials (over 3 hours) form</a>
+          or <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">unused materials (over 3 hours) form</a>
```
Comment (naseberry) [inline config/locales/en.yml]: @jsugarman ps - you don't need the title attribute as the link text is almost the same and already descriptive.
Need a visually hidden text to inform AT users of a new window.

File: config/locales/en.yml [hunk 2]
```diff
@@ -1450,7 +1450,7 @@ en:
             clar_page_intro_html: |
               If needed, please complete
               <a href="https://www.gov.uk/government/publications/lf1-claim-litigator-fees" target="_blank" rel="external" title="Special Prep Form">special prep form</a>
-              and/or <a href="https://www.gov.uk/government/publications/lf1-claim-litigator-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">Unused materials (over 3 hours) form</a>
+              or <a href="https://www.gov.uk/government/publications/lf1-claim-litigator-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">unused materials (over 3 hours) form</a>
```
Comment (naseberry) [inline config/locales/en.yml]: ditto

File: config/locales/en.yml (hunk 1 above)
Comment (jsugarman) [reply config/locales/en.yml]: this is just a minor content change to existing stuff, can you add a commit for the visually hidden stuff or put an example in here

**Review by naseberry — APPROVED** [review]
👍

**Comment by jsugarman** [comment]
closed in favour of https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3486

**Pull request closed without merging**